### PR TITLE
Global style

### LINKED ISSUE #29: Global Responsive screening
# User Story
As a **user**,
I want to **have responsive layout**,
so I can **view the app on diffrent screens**.

# Description

#Material

# Done

- [x] Remove button border :focus / :active

### PR DESCRIPTION
**Pull request by LebkuchenCJ**
Fix #29 

## Comments: 
 - Thought about create an app view effect on window screening but cxl that idea throughout the progress
 - Fixed some small global issues as setting margin globally

### CODE REVIEW

**Comment by JonasImm** [comment]
Button. {
Cursor: pointer
}

File: client/src/GlobalStyles.js [hunk 1]
```diff
@@ -37,6 +37,27 @@ function GlobalStyles() {
...
+          a {
+            cursor: pointer;
+          }
+
+          h2,
+          h3,
+          h4,
+          h5 {
+            margin: 0;
+          }
```
Comment (DrBennemin) [inline client/src/GlobalStyles.js:57]: I suggest putting in a linebreak in this line for a better readability and consistency <3

**Review by DrBennemin — CHANGES_REQUESTED** [review]
Looks very good to me. just a single suggestion :)!

File: db.json [hunk 2]
```diff
@@ -27,6 +27,10 @@
     {
       "name": "Wurst",
       "id": 7
+    },
+    {
+      "name": "",
+      "id": 8
```
Comment (JoscaWij) [inline db.json:33]: Do you need this entry to show something? If not I would delete the entry or revert the commit 😸

**Review by JoscaWij — APPROVED** [review]
nice job 🚀 lgtm

just one minor thing: 
I would call the font-title-main -> font-color-title and font-color-welcome. because it is still a color and not a font - with font-title-main I would have expected a font 😄

File: db.json (hunk 2 above)
Comment (LebkuchenCJ) [reply db.json:33]: Oh noooo. Too late. I already merged it. :(

**Pull request merged**